examples/tailwindcss/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
@juwiragiye Can you please delete the Tailwind CSS example from this branch/PR?

Reply:
Comment